### ISSUE TITLE
Auto-getter methods missing from `source_map.pc_pos_map`

### ISSUE THREAD

**Issue by antazoey** [vyperlang/vyper#3271]
### Version Information

* vyper Version (output of `vyper --version`): 0.3.7
* OS: macOS
* Python Version (output of `python --version`): Python 3.9.13

### What's your issue about?

Auto-getter methods from contracts are missing from the PCMap.
To reproduce, create a simple contract like this:

```vyper
# @version 0.3.7

owner: public(address)
```

The output when compiling (assume var named `result`) shows as this:  `result["source_map"]["pc_pos_map"]` is `{}`, so it seems missing?

### How can it be fixed?

Not sure!

**Comment by charles-cooper** [comment]
this is fixed in master (as of 4f9f8133191c25841e309ad26f3124dbe2a46b21):
```bash
$ vyc -f source_map repros/3271.vy 
{"breakpoints": [], "error_map": {"39": "fallback function"}, "pc_breakpoints": [], "pc_jump_map": {"0": "-", "10": "-", "21": "-", "33": "-", "40": "-"}, "pc_pos_map": {"11": [1, 14, 1, 21], "16": [1, 14, 1, 21], "17": [1, 14, 1, 21], "18": [1, 14, 1, 21], "21": [1, 14, 1, 21], "30": [1, 14, 1, 21], "32": [1, 14, 1, 21], "33": [1, 14, 1, 21]}, "pc_pos_map_compressed": "-1:-1:0:-;;;;;;:::-;14:7;;;;:::-;-1:-1;;;;;14:7;;:::-;-1:-1;;;;:::-;;;"}
```

**Comment by antazoey** [comment]
I have verified that the issue is fixed, thank you.
I tested on both `vyper_json` and the vyper CLI compile source via vvm.
